maths/fibonacci.py

```diff
@@ -83,6 +83,29 @@ def fib_iterative(n: int) -> list[int]:
     return fib
 
 
+def fib_recursive_term(i: int) -> int:
+    """
+    Calculates the i-th (0-indexed) Fibonacci number using recursion
+    >>> fib_recursive_term(0)
+    0
+    >>> fib_recursive_term(1)
+    1
+    >>> fib_recursive_term(5)
+    5
+    >>> fib_recursive_term(10)
+    55
+    >>> fib_recursive_term(-1)
+    Traceback (most recent call last):
+        ...
+    Exception: n is negative
+    """
```

Comment:
Can we:
1. Removed the large block of empty lines in the middle of the fib_recursive function.
2. Ensured the docstring for the fib_recursive function is correctly formatted and includes the expected outputs for the doctests.